python/sparkdl/utils/jvmapi.py

```diff
@@ -0,0 +1,51 @@
+#
```

Comment:
is this module basically meant to be exposed only through `utils.pyutils` below?  might want to add a comment.